### ISSUE TITLE
SD2Vita won't show up whatsoever in vitashell anymore after I hot swapped it

### ISSUE THREAD

**Issue by alex1256783** [CelesteBlue-dev/PSVita-StorageMgr#80]
Pretty much the title. I was an idiot and made the mistake of hot swapping the sd2vita without realizing the consequences and now my micro sd won't appear as my ux0 storage but instead my old 8gb memory card. Obviously it was a huge mistake to hot swap it and I realize that now, but is there any way for me to make it work again without my brand new and expensive micro sd going to complete waste? Thanks for any help

**Comment by ghost** [comment]
(⊙_⊙;)

**Comment by ghost** [comment]
Oh...

**Comment by alex1256783** [comment]
> Oh...

Any ideas?

**Comment by Duckbuster** [comment]
Does the microsd still get read by a computer?

**Comment by hedhehd** [comment]
Nope. (¬_¬ )

**Comment by ghost** [comment]
> > Oh...
> 
> Any ideas?

Nope. :-(

**Comment by ghost** [comment]
Sorry

**Comment by alex1256783** [comment]
> Does the microsd still get read by a computer?

Yes, when I put the micro sd into my reader and then plug it into my computer it immediately reads on my computer

**Comment by alex1256783** [comment]
> Sorry

No worries, thank you for taking the time to read

**Comment by ghost** [comment]
Well, now whos gonna say the idea? Because i dont know.

**Comment by alex1256783** [comment]
> Well, now whos gonna say the idea? Because i dont know.

At this point, I just want to know what's having the issue so I don't needlessly buy a replacement of something that still works. Like if it's my sd2vita then I can save myself buying another micro sd card

**Comment by DasKatzchen** [comment]
If you can still read it in a PC, I'd back up everything from it, reformat it with zzblank as if it were a new card, and then put everything back and try it again.

**Comment by alex1256783** [comment]
> If you can still read it in a PC, I'd back up everything from it, reformat it with zzblank as if it were a new card, and then put everything back and try it again.

Unfortunately I've tried that 3 plus times, and everything goes the exact same as the first time until I open vitashell at the very end and there is no extra storage (the first time I did it showed my 476gb in ux0). The only difference from the first time I did all that is when copying my backed up vita files onto it... the first time, when it worked, it took maybe 10-20 minutes to copy all the files over. The next and every time afterwards, took 2+ hours to copy over
